### DISCUSSION TITLE
Hotkeys for multiple terminals?

### DISCUSSION THREAD

**Discussion by onichandame** [lapce/lapce#1729] [Q&A]
As a (former) VSCode user, I'm used to working with multiple terminals at a time only with keyboard. One example case may be one terminal is blocked by a build process and I need to quickly do something else on another one. Common functions I use through hotkeys are:
- create new terminal
- move to next(previous) terminal

In Lapce I only see a 
![image](https://user-images.githubusercontent.com/23728505/202902488-51f42a88-2cf1-47ec-9d6b-536cbae6cd00.png)
 button at the topright of the terminal panel. Searching for `terminal` in the keybinding settings shows nothing I need. Is it a feature not implemented yet?

**Comment by MinusGix** [comment]
There's a command to focus the terminal, but there's no commands for creating a new terminal or for swapping focus between terminals. However, this should be easy to add.

**Comment by igotfr** [comment]
there are a plus button to open a new terminal
